core/src/main/java/io/grpc/Context.java

```diff
@@ -627,17 +623,17 @@ public boolean isCurrent() {
 
     /**
      * Attach this context to the thread and return a {@link AutoCloseable} that can be
-     * used with try-with-resource statements to properly {@link #detach} and {@link #cancel}
-     * the context on completion.
+     * used with try-with-resource statements to properly attach the previously bound context
```

Comment:
Use the old JavaDoc for this method (it mentioned detach instead of "attach the previous")?